tests/model_serving/model_runtime/mlserver/gpu/test_mlserver_gpu_serving.py

```
@@ -0,0 +1,174 @@
+    ) -> None:
+        """Verify model readiness, server health, and REST inference via external route.
+
+        Given an ISVC with mlserver-cuda-runtime, nvidia.com/gpu: 1, external_route=True
+        When readiness, health, and inference endpoints are queried
+        Then all return HTTP 200 with valid response structure.
+        """
+        base_url = get_exposed_isvc_url(isvc=mlserver_inference_service)
+        model_name = mlserver_inference_service.name
+
+        readiness_url = f"{base_url}/v2/models/{model_name}/ready"
+        readiness_response = requests.get(url=readiness_url, verify=False, timeout=60)
```

Ruff (0.16.0)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)